scripts/build_ffmpeg.sh

```diff
@@ -0,0 +1,86 @@
+# !/bin/bash
+
+if [ "$#" -lt 2 ]; then
+	echo "usage build_ffmpeg <ffmpeg build path> <product path> <release/debug - optional>"
+	exit
+fi
+
+FFMPEG_BUILD_PATH=$1
+PRODUCT_ROOT_PATH=$2
+ADDON_BUILD_PATH=$PRODUCT_ROOT_PATH/node_addons/FormatConverter/build/
+FFMPEG_SYMLINK=$ADDON_BUILD_PATH/FFmpeg
+BUILD_CONF=
+OS=`uname`
+
+echo "PRODUCT_ROOT_PATH=$PRODUCT_ROOT_PATH"
+
+[ -d $FFMPEG_BUILD_PATH ] || mkdir -p "$FFMPEG_BUILD_PATH"
+
+[ "$#" -eq 3 ] && BUILD_CONF=$3 || BUILD_CONF=Release
+
+if [ "$BUILD_CONF" = "debug" ]; then
+	BUILD_CONF=Debug
+	echo "Debug mode"
+else
+	BUILD_CONF=Release
+	echo "Release mode"
+fi
+
+mkdir -p ${BUILD_CONF}
+
+
+echo "current path `pwd`"
+echo "FFMPEG_BUILD_PATH=$FFMPEG_BUILD_PATH"
+echo "ADDON_BUILD_PATH=$ADDON_BUILD_PATH"
+
+echo "unlink $FFMPEG_SYMLINK"
+unlink $FFMPEG_SYMLINK
+
+
+if [ ! -d $FFMPEG_SYMLINK ]; then
+
+    curl -OL https://github.com/FFmpeg/FFmpeg/releases/download/n3.0/ffmpeg-3.0.tar.gz
```

Comment:
Set FFMPEG_VERSION and use it rather than repeating the literal '3.0' six times. Much easier to maintain.